fastn.com/podcast/fastn-p2p-emails.ftd

```diff
@@ -0,0 +1,219 @@
+leveraging existing email clients, so on day one we're as feature-complete as
+any regular email client. Whether Apple Mail is better than Gmail's interface
+is subjective, but we're not losing functionality.
+
+Of course, our mail server needs to become more sophisticated - separate
+mailboxes per username, backup systems, retry logic for failed deliveries, and
+spam handling.
+
+-- ds.h2: Web Interface Option
+
+fastn mail can also expose an HTTP service through fastn-net's bridge
+functionality, providing a web mail interface. This would use the same
+username/password authentication as SMTP, allowing users to access their email
+without needing a dedicated client.
+
+We could integrate existing web mail solutions, though I'd prefer keeping
+everything in Rust and potentially using fastn for the web interface to
+maintain consistency.
+
+-- ds.h2: Extending to Chat: fastn Chat
+
+The same concept could apply to chat. We could create fastn chat that works
+with existing chat clients like Pidgin, which has a plugin infrastructure for
+connecting to multiple chat services.
+
+Unlike email, there's no standard chat protocol, so we'd be creating our own.
+But we could still leverage existing clients through plugins, and our fastn-net
+bridge could provide a basic web chat interface.
+
+We could even create Discord bot bridges - any message sent on a Discord
+channel could automatically be mirrored to peer-to-peer chat and vice versa.
+
+-- ds.h2: Strategic Positioning
+
+This approach solves a fundamental problem: in our current centralized
+internet, everything depends on SaaS services. Mail is one of the most critical
+dependencies. Making mail distributed and peer-to-peer is a significant step
+toward digital independence.
+
+fastn is about building a decentralized, peer-to-peer world from the ground up.
+This requires a good programming language that ordinary people can use, a great
+platform, auto-merge capabilities, and now mail functionality.
+
+-- ds.h2: Launch Strategy
+
+Instead of waiting to complete Pista (our collaborative document platform),
+which requires auto-merge, identity management, and relationship handling,
+fastn mail only requires POP and SMTP implementation. It has revenue potential
+and addresses a real need.
+
+We can position this as fastn mail rather than creating yet another brand. This
+fits into the broader fastn ecosystem - we're building a comprehensive platform
+for the decentralized web.
+
+-- ds.h2: Development Plan
+
+**Siddhant**: This gives me a good coding project and content creation
+opportunity. I can work on fastn mail while you continue with Pista, and
+document the development process through blog posts and videos.
+
+**amitu**: Exactly. Let's build fastn mail in public. We can release this
+conversation as our next podcast episode, then create a series documenting the
+development process. Every coding session, every design decision - let's record
+it all.
```

Comment:
Should we try to do more non podcast recordings?

Reply:
I was going to try this. The code in https://github.com/fastn-stack/fastn/pull/2181/files was supposed to be written on a live stream but I did not do it because that would have taken more time. I think demo style videos (like [this one](https://youtu.be/WV8ZOZ4mbCI) or [this](https://youtu.be/BT9QuvR48vc) would be better for viewers (and me).